rustls/src/anchors.rs

```diff
@@ -102,6 +102,10 @@ impl RootCertStore {
     }
 
     /// Add a single DER-encoded certificate to the store.
+    ///
+    /// This is suitable for a small set of root certificates that are expected to parse
```

Comment:
I'm nitpicking, but I feel like this is pretty opaque language. I'd want to know how this is different in order to understand when/why I use one or the other.

Reply:
What about if I tweaked the final sentence from:
> "If you're adding a larger collection of certificates, for instance from a system root store, prefer [`RootCertStore::add_parsable_certificates`]."

to something like:

> "If you're adding a larger collection of certificates, for instance from a system root store, prefer [`RootCertStore::add_parsable_certificates`] to allow ignoring malformed roots without surfacing 
an error."

I think that "ignore bad roots and add what you can" is the difference to call out.

Reply:
But how is that ultimately different? It's kind of trivial to ignore the error from this function, too, right?

Reply:
That's a fair point. Do you think there's any utility in `add_parseable_certificates` from that perspective?

Reply:
@djc Take a look at https://github.com/rustls/rustls/pull/1249. If that's still not to your liking I can revert the original commit and we can go back to the drawing board.